### ISSUE TITLE
bug: Missing entry_point.txt in localstack installed via pip

### ISSUE THREAD

**Issue by egobrain** [localstack/localstack#6545]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

```
$ pip3 install 'locastack[full]'
$ ls python3.10/site-packages/localstack-1.0.2.dist-info/entry_points.txt
python3.10/site-packages/localstack-1.0.2.dist-info/entry_points.txt: No such file or directory
```

Without this file it's impossible to run localstack in host mode (for debug and development) because there is no services with `default` provider (only `pro` and `pro_legacy` exists)

### Expected Behavior

localstack-*.dist-info/entry_points.txt exists

### How are you starting LocalStack?

With the `localstack` script

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    pip3 install 'localstack[full]'
#### or
    pip3 install 'localstack[runtime]' 

    localstack start --host

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal sqs create-queue --queue-name sample-queue 

### Environment

```markdown
- OS: debian
- LocalStack: 1.0.2
```

**Comment by dfangl** [comment]
Hi!
Thank you for your contribution. You are right that its currently missing, we have identified this issue just yesterday and already fixed it in the release pipeline, meaning it will be fixed on the next release of the `locakstack` package, which should happen failry soon.

**Comment by dfangl** [comment]
Hi!
This issue should be resolved with pip release 1.0.3, as you can see here:
```
$ tar tvf localstack-1.0.3.tar.gz | grep "entry_points.txt"
-rw-r--r-- runner/docker   2717 2022-07-30 02:29 localstack-1.0.3/localstack.egg-info/entry_points.txt
```
